tests/components/sensorpush_cloud/conftest.py

```diff
@@ -0,0 +1,62 @@
+"""Common fixtures for the SensorPush Cloud tests."""
+
+from __future__ import annotations
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+import pytest
+from sensorpush_ha import SensorPushCloudApi, SensorPushCloudHelper
+
+from homeassistant.components.sensorpush_cloud.const import DOMAIN
+from homeassistant.const import CONF_EMAIL
+
+from .const import CONF_DATA, MOCK_DATA
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_api(request: pytest.FixtureRequest) -> Generator[AsyncMock]:
+    """Override SensorPushCloudApi."""
+    mock_api = AsyncMock(SensorPushCloudApi)
+    with (
+        patch(
+            "homeassistant.components.sensorpush_cloud.config_flow.SensorPushCloudApi",
+            return_value=mock_api,
+        ),
+    ):
+        yield mock_api
+
+
+@pytest.fixture
+def mock_helper(request: pytest.FixtureRequest) -> Generator[AsyncMock]:
+    """Override SensorPushCloudHelper."""
+    mock_helper = AsyncMock(SensorPushCloudHelper)
+    mock_helper.async_get_data.return_value = MOCK_DATA
+    with (
+        patch(
+            "homeassistant.components.sensorpush_cloud.coordinator.SensorPushCloudHelper",
+            return_value=mock_helper,
+        ),
+    ):
+        yield mock_helper
```

Comment:
```suggestion
def mock_helper(request: pytest.FixtureRequest) -> Generator[MagicMock]:
    """Override SensorPushCloudHelper."""
    with (
        patch(
            "homeassistant.components.sensorpush_cloud.coordinator.SensorPushCloudHelper",
             autospec=True,
        ) as mock_helper,
    ):
    	helper = mock_helper.return_value
    	helper.async_get_data.return_value = MOCK_DATA
        yield helper
```
is the more standard approach imo